### PR TITLE
Update docker file

### PR DESCRIPTION
**Pull request by chb0github**
I updated the `Dockerfile` which apparently hasn't been updated in 4 years. The base image provides a functioning working environment, but the `goreleaser` is having issues. 

I needed this so I could use it with JetBrains spaces.

### CODE REVIEW

**Review by danielmkn — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-
-ENV GO111MODULE=on
-
-COPY go.mod .
-COPY go.sum .
-
+FROM alpine AS base
+RUN apk add --no-cache git terraform wget make && \
+    wget -q https://github.com/goreleaser/goreleaser/releases/download/v1.21.2/goreleaser_1.21.2_x86_64.apk  && \
+    wget -q https://go.dev/dl/go1.21.1.linux-amd64.tar.gz && \
```
Comment (alexhung) [inline Dockerfile:4]: FYI, we are still using go 1.19 because JFrog Pipeline only support 1.19 runtime.

**Comment by chb0github** [comment]
If you really want, I'll downgrade it - but since no one was using it, and the default in alpine is like 1.16, I just went for broke

**Comment by chb0github** [comment]
Give me a moment and I can downgrade it

**Comment by alexhung** [comment]
> If you really want, I'll downgrade it - but since no one was using it, and the default in alpine is like 1.16, I just went for broke

It's more of using this Dockerfile and people adding 1.20/1.21 specific code that we can't compile later on.

**Comment by chb0github** [comment]
It's now downgraded to 1.19.2

**Review by alexhung — APPROVED** [review]

**Comment by chb0github** [comment]
Yeah, I accidentally deleted the repo when i was getting funny behavior and reforked it. I'll create a new PR from the same code

**Pull request closed without merging**